hydra-configs-torchvision/tests/test_instantiate_transforms.py

```diff
@@ -297,6 +298,6 @@ def test_instantiate_classes(
     full_class = f"hydra_configs.torchvision.{modulepath}.{classname}Conf"
     schema = OmegaConf.structured(get_class(full_class))
     cfg = OmegaConf.merge(schema, cfg)
-    obj = instantiate(cfg, *passthrough_args, **passthrough_kwargs)
+    obj = instantiate(cfg, *passthrough_args, **passthrough_kwargs, _convert_="all")
```

Comment:
that's a big hammer.
It may be reasonable for pytorch classes, but you are enabling it for all of them at the same time.
you can also add it to the cfg object for this test.

Reply:
migrated `_convert_='all'` to the 4 offending configs.